src/modules/NetworkLogger.ts

```diff
@@ -2,14 +2,28 @@ import type { RequestHandler } from '@apollo/client';
 
 import InstabugConstants from '../utils/InstabugConstants';
 import xhr, { NetworkData, ProgressCallback } from '../utils/XhrNetworkInterceptor';
-import { reportNetworkLog, isContentTypeNotAllowed } from '../utils/InstabugUtils';
+import { isContentTypeNotAllowed, reportNetworkLog } from '../utils/InstabugUtils';
+import { NativeModules } from 'react-native';
 
 export type { NetworkData };
 
 export type NetworkDataObfuscationHandler = (data: NetworkData) => Promise<NetworkData>;
 let _networkDataObfuscationHandler: NetworkDataObfuscationHandler | null | undefined;
 let _requestFilterExpression = 'false';
 
+const filterURL = (url: string): string => {
+  const [protocol, rest] = url.split('://');
+  return (rest || protocol).split('/')[0];
+};
+
+const getDevServerURL = (): string | null => {
+  const { scriptURL } = NativeModules.SourceCode;
+  if (scriptURL) {
+    return filterURL(new URL(scriptURL).toString());
```

Comment:
Why do we need to use `new URL(..).toString()`? Does it do any kind of normalization for the `scriptURL`?

Reply:
fixed